mon-pix/app/styles/components/_tutorial-card.scss

```
@@ -46,39 +46,75 @@
   }
 
   &__actions {
+    justify-content: space-between;
     align-items: end;
     display: flex;
     flex: 1;
     gap: 13px;
   }
 }
 
-.tutorial-card-v2-content-actions {
+.pix-action-chip {
+  width: 36px;
+  height: 36px;
+  background: $white;
+  border-radius: 50%;
+  border: 1px solid $grey-10;
+  text-align: center;
 
-  &__save,
-  &__evaluate {
-    font-size: 0.813rem;
-    font-weight: $font-medium;
+  &-icon {
+    left: calc(50% - 20px / 2);
+    top: 0%;
+    bottom: 0%;
     color: $grey-50;
-    letter-spacing: 0.028rem;
-    background: transparent;
-    border: none;
-    padding: 0;
-    display: block;
-
-    svg {
-      margin-right: 4px;
+    font-size: 18px;
+  }
+
+  &:hover {
+    cursor: pointer;
+    border: 1.4px solid $grey-50;
+    background: $grey-10;
+  }
+
+  &:disabled {
+    cursor: initial;
+    pointer-events: none;
+  }
+
+  &.pix-action-chip-selected {
```

Le `selected` est un Modifier dans le convention [BEM](http://getbem.com/naming/), il faut donc le précéder d'un double trait d'union. 
```suggestion
  &.pix-action-chip--selected {
```